rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -26,13 +26,18 @@
 
 
 class PyPIURLFixHook(BaseSpecHook):
-    """
-    SpecHook for transforming old nonfunctional python package url to
-    a safe url(eg. https://pypi.* to https://files.pythonhosted.*)
+    """SpecHook for transforming an old python package url to a new url(eg.
```

Comment:
While at it, how about moving the opening bracket to the next line (there is missing space anyway), and capitalizing URL (I don't really understand why it's lowercase now)?